packages/RadioCard/RadioCard.jsx

```diff
@@ -0,0 +1,225 @@
+  '&:hover': {
+    transform: 'scale(1.025)',
+  },
+  '@media (prefers-reduced-motion: reduce)': {
+    transition: 'none !important',
+    transform: 'none !important',
+  },
+  [`${HiddenInput}:focus ~ & > div > div > ${FakeRadio}`]: {
+    boxShadow: `0 0 4px 1px ${colorGreyShuttle}`,
+    borderColor: colorWhite,
+  },
+  [`${HiddenInput}:checked ~ & > div > div > ${FakeRadio}`]: {
+    '& > span': {
+      display: 'block',
+    },
+    borderColor: colorGreyShuttle,
+  },
+})
+
+const InnerChecked = styled.span({
+  width: '0.75rem',
+  height: '0.75rem',
+  borderRadius: '50%',
+  backgroundColor: colorAccessibleGreen,
+  display: 'none',
+})
+
+const labelBox = {
+  width: '100%',
+  marginRight: '0.5rem',
+  '@media (min-width: 768px)': {
+    marginRight: '1rem',
+  },
+}
+const getGeneratedId = (name, value) => {
+  return generateId(name).postfix(value)
+}
+
+const StyledLabelBox = styled(Box)(labelBox)
+const StyledChildrenBox = styled(Box)({ marginLeft: '2.25rem' })
+const StyledDiv = styled.div(({ fullHeight }) => {
+  if (fullHeight) {
+    return { height: '100%' }
+  }
+  return {}
+})
+
+/**
+ * @version ./package.json
+ * @visibleName RadioCard (beta)
+ */
+const RadioCard = React.forwardRef(
+  ({ id, name, value, label, fullHeight, variant, children, ...rest }, ref) => (
+    <StyledDiv fullHeight={fullHeight}>
+      <HiddenInput
+        type="radio"
+        id={id || getGeneratedId(name, value)}
+        name={name}
+        value={value}
+        data-testid="hidden-input"
+        variant={variant}
+        ref={ref}
+        {...safeRest(rest)}
+      />
+
+      <StyledLabel
+        htmlFor={id || getGeneratedId(name, value)}
+        variant={variant}
+        data-testid="checkbox-label"
+      >
+        <StyledLabelBox vertical={4} horizontal={3}>
+          <Box between={3} inline>
+            <FakeRadio data-testid="fake-input">
+              <InnerChecked />
+            </FakeRadio>
+            <Heading level="h3">{label}</Heading>
+          </Box>
+          {children && <StyledChildrenBox>{children}</StyledChildrenBox>}
+        </StyledLabelBox>
+      </StyledLabel>
+    </StyledDiv>
+  )
+)
+
+RadioCard.propTypes = {
+  /**
+   * The style.
+   */
+  variant: PropTypes.oneOf(['standard', 'brand']),
+  /**
+   * The label.
+   */
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+  /**
+   * Associate this radio card with a group. Set as the HTML name attribute.
+   */
+  name: PropTypes.string.isRequired,
+  /**
+   * The value. Must be unique within the group.
+   */
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.bool]).isRequired,
+  /**
+   * Use `checked` for controlled radio.
+   */
+  checked: PropTypes.bool,
+  /**
+   * The id. Must be unique within the group. If no id is provided, one will be generated in the following format: `name_value`
+   */
+  id: PropTypes.string,
+  /**
+   * Sets the `Card`'s `height` equal to its parent.
```

Comment:
Can you change this to ...the RadioCard's height...

Reply:
Done :)